services/121-service/src/payments/reconciliation/intersolve-voucher-reconciliation/intersolve-voucher-reconciliation.controller.ts

```diff
@@ -59,6 +59,7 @@ export class IntersolveVoucherReconciliationController {
   })
   @Patch('/financial-service-providers/intersolve-voucher/unused-vouchers')
   public async cronRetrieveAndUpdatedUnusedIntersolveVouchers(): Promise<void> {
+    // ##TODO: What is the reason again we cannot log the console message in the CronjobService itself?
```

Comment:
Because cronjob api requests can have a timeout

Reply:
I will remove this comment.